src/createServer.js

```diff
@@ -1,3 +1,80 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase.js');
+
+const SUPPORTED_CASES = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const url = new URL(req.url, 'http://localhost:5700');
+    const textToConvert = url.pathname.slice(1); // видаляємо початковий "/"
+    const targetCase = url.searchParams.get('toCase');
+
+    const errors = [];
+
+    // Валідація параметрів
+    if (!textToConvert) {
+      errors.push({
+        message:
+          // eslint-disable-next-line max-len
+          'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    if (!targetCase) {
+      errors.push({
+        message:
+          // eslint-disable-next-line max-len
+          '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    } else if (!SUPPORTED_CASES.includes(targetCase)) {
+      errors.push({
+        message: `This case is not supported. Available cases: ${SUPPORTED_CASES.join(', ')}.`,
+      });
+    }
+
+    // Якщо є помилки, відповідаємо з кодом 400
+    if (errors.length > 0) {
+      res.writeHead(400, {
```

Comment:
Consider adding a status text 'Bad Request' to the response header when validation fails. This aligns with the task requirement to respond with a `Bad Request` status text for invalid requests.